pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -37,8 +38,8 @@ const (
 	bottlerocketUserData     = `
 [settings.kubernetes]
 api-server = "{{.Cluster.Endpoint}}"
-cluster-certificate = "{{.Cluster.CABundle}}"
-cluster-name = "{{.Cluster.Name}}"
+cluster-certificate = "{{if .Cluster.CABundle}}{{.Cluster.CABundle}}{{end}}"
+cluster-name = "{{if .Cluster.Name}}{{.Cluster.Name}}{{end}}"
```

Comment:
Just double checking that if Cluster.Name = ptr.String(""), then this if statement will return true and add this line.